src/components/AccountForms/RegistrationForm/index.js

```diff
@@ -204,20 +202,40 @@ const RegistrationForm = ({ mockOnSubmit }) => {
     retypePassword: false,
   });
   const history = useHistory();
+  // const [error, setError] = useState({ userName: { message: '' } });
+
+  /*const key = Object.keys(error || {}).toString();
+  if (key === 'email') {
+    const { email } = error;
+    errors.email = email;
+  }*/
 
   const onSubmit = (data) => {
-    request.post(API_PATH.REGISTER, data).then(({ status }) => {
-      if (status === 200) {
-        const inProd = process.env.NODE_ENV === 'production';
-        const options = {
-          category: 'onSubmit',
-          action: 'Created an Account',
-        };
-        inProd && event(options);
-        dispatch(updateAuth(auth.checkCookie()));
-        history.push('/mentors');
-      }
-    });
+    request
+      .post(API_PATH.REGISTER, data)
+      .then(({ status }) => {
+        if (status === 200) {
+          const inProd = process.env.NODE_ENV === 'production';
+          const options = {
+            category: 'onSubmit',
+            action: 'Created an Account',
+          };
+          inProd && event(options);
+          dispatch(updateAuth(auth.checkCookie()));
+          history.push('/mentors');
+        }
+      })
+      .catch(({ err }) => {
+        const message = _get(
```

Comment:
with the latest update in javascript land, lodash get became obsolete.

you can do `object?.something` where the question mark is saying "only do object.something if object is defined"

you can set a fallback by doing `object?.something || yourDefaultThing` otherwise by default it would return undefined

Reply:
done